docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -97,26 +97,22 @@ The Telemetry module supports the convenient shipment of applications and access
       </details>
     </div>
 
-2. Wait for the LogPipeline to be in the `Running` state. To check the state, run:
-
-    ```bash
-    kubectl get logpipelines
-    ```
+2. Wait for the LogPipeline to be in the `Running` state. To check the state, run `kubectl get logpipelines`.
 
 ### Set Up Access Logs
 
 By default, Istio sidecar injection and Istio access logs are disabled in Kyma.
 
 To analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime, you must enable them:
```

Comment:
Minor; however, in DITA CCMS these to paragraphs are combined. We may want to stick to the same formatting.

Reply:
agreed!